src/test/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtilTest.java

```diff
@@ -132,33 +134,42 @@ public void shouldReturnTrueValidateOrgIdentifier() {
     }
 
     @Test
-    public void mapUserInfoCorrectly() {
-        ProfessionalUser professionalUser = new ProfessionalUser();
-        professionalUser.setFirstName("abc");
-        professionalUser.setLastName("bcd");
-        professionalUser.setEmailAddress("a@b.co.uk");
-        professionalUser.setIdamStatusCode("200");
-        professionalUser.setIdamMessage("Success");
-
-        ProfessionalUser mappedUser = RefDataUtil.mapUserInfo(professionalUser, mock(ResponseEntity.class), true);
-        assertThat(mappedUser).isNotNull();
+    public void mapUserInfoCorrectly_with_roles() {
+
+        HttpHeaders header = new HttpHeaders();
+        header.setContentType(APPLICATION_JSON);
+        ResponseEntity<?> realResponseEntity = new ResponseEntity<>(getUserProfileResponse, header, HttpStatus.OK);
+
+        ProfessionalUser responseUser = RefDataUtil.mapUserInfo(professionalUser, realResponseEntity, true);
```

Comment:
could you please prefix test before test case?

Reply:
Done.